torchgeo/trainers/byol.py

```diff
@@ -308,66 +309,74 @@ def update_target(self) -> None:
 
 
 class BYOLTask(pl.LightningModule):
-    """Class for pre-training any PyTorch model using BYOL."""
+    """Class for pre-training any PyTorch model using BYOL.
+
+    Supports any available `Timm model
+    <https://rwightman.github.io/pytorch-image-models/>`_
+    as an architecture choice. To see a list of available pretrained
+    models, you can do:
+
+    .. code-block:: python
+
+        import timm
+        print(timm.list_models(pretrained=True)
```

Comment:
Same here, I would suggest listing all available models:
```suggestion
        print(timm.list_models())
```
Also, you were missing a parenthesis